dragonflybsd/libs.md

```diff
@@ -0,0 +1,105 @@
+# Instructions for obtaining EFI related libraries on DragonflyBSD
+
+Read [README.md](./README.md) first for general information.
+
+DragonflyBSD already has support for EFI variables in kernel, so no need to use
+a custom one.
+
+The plan is:
+ * prepare environment for building ports
+ * build and install `efivar-bsd` port (might be renamed to `efivar` later)
+ * build a test tool
+ * verify that it works
+
+## Preparing for using ports
+
+`git`, `gcc` and other basic tools should be installed by default, so we only
+need to pull ports tree. Run this and other commands below as a `root` user:
+
+```
+git clone --depth 1 -b efivar https://github.com/3mdeb/DPorts.git /usr/ports
```

Comment:
@SergiiDmytruk There is something wrong with the paths - either here or in the ports. Should it be `/usr/dports` here instead? If i change to `dports` (here and below) it works.

Reply:
Sure, I copied and edited OpenBSD's file and missed this.